test/integration/infrastructure/infrastructure_test.go

```diff
@@ -149,61 +151,143 @@ var _ = Describe("Infrastructure tests", func() {
 	})
 
 	Context("with infrastructure that requests existing vpc", func() {
-		var (
-			identifiers infrastructureIdentifiers
-		)
+		It("should successfully create and delete", func() {
+			identifiers := prepareVPC(ctx, clientFactory, *region, vpcCIDR, natGatewayCIDR)
+			defer func() {
+				cleanupVPC(ctx, clientFactory, identifiers)
```

Comment:
It will be better to put in clean up action. see here. https://github.com/gardener/gardener-extension-provider-aws/blob/master/test/integration/infrastructure/infrastructure_test.go#L205